autocertkit/network_tests.py

```diff
@@ -1033,7 +1033,7 @@ def validate_multicast(self):
             raise TestCaseError('Error: Multicast test failed. iperf data: %s'
                                 % cmd_result)
 
-        self.iperf_data = self.parse_iperf_line(cmd_result)
```

Comment:
This means the xml will no longer have the iperf data stored in it doesn't it?

Reply:
Sorry I am not aware of my reply disappeared, but I think you should already received github notification mail.
If not please let me know, I will re-add.

Reply:
Yes, it's Mutlitcast test bug.

Reply:
So where will the iperf data be kept? We need to record this output.

Reply:
It did not record in anyplace, so I said it's a bug when Multicast test class coding.
The data should be 'return'-ed then test method will record to xml file. I corrected it here.